raiden-dapp/tests/e2e/utils/blockchain-interaction.ts

```diff
@@ -88,6 +88,9 @@ export function withdrawUDCTokens() {
   cy.getWithCustomTimeout('[data-cy=udc_withdrawal_dialog]').should('not.exist');
 }
 
+/**
+ *
```

Comment:
These comments have no comments? :) Did you forget to add a comment?

Reply:
Yes totally  :)

Reply:
you would take care about it @taleldayekh ?

Reply:
Sure, don't worry about it. 👍

Reply:
There seems to be a rule that requires `JSDoc comments` to be added. When I remove them my editor adds them back again on save.